--- .local/lib/node_modules/svg-sankey/index.es5.js	2019-05-20 17:14:08.450097223 -0600
+++ .local/lib/node_modules/svg-sankey/index.es5.js	2019-05-20 17:13:38.177881032 -0600
@@ -116,7 +116,7 @@
 
   parts.push(sourceTitle + ' \u2192 ' + targetTitle);
   if (matTitle) parts.push(matTitle);
-  parts.push(fmt(d.data.value));
+  parts.push(fmt(d.value));
   return parts.join('\n');
 }
 
@@ -131,7 +131,7 @@
   var ordering = data.metadata && data.metadata.layers ? data.metadata.layers : data.order && data.order.length ? data.order : null;
 
   var layout = (0, _d3SankeyDiagram.sankey)().linkValue(function (d) {
-    return d.data.value;
+    return d.value;
   }).size([width - margins.left - margins.right, height - margins.top - margins.bottom]).ordering(ordering).rankSets(data.rankSets);
 
   if (_commander2.default.position) {
